addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -39,14 +54,24 @@ The binding has the following configuration options, which can be set for "bindi
 The Onkyo AVR thing requires the ip address and the port to access it on.
 In the thing file, this looks e.g. like
 
-`onkyo:onkyoAV:myOnkyo [ipAddress="192.168.1.100", port="60128"]`
+Model specific
+`onkyo:TX-NR818:myOnkyo [ipAddress="192.168.1.100", port="60128"]`
```

Comment:
please add a new line above the code section and use three backticks for escaping the code